tests/testthat/test-ggMarginal.R

```diff
@@ -1,52 +1,28 @@
+expectDopp2 <- function(testName, funName, ggplot2Version) {
```

Comment:
do you remember what this "Dopp" stands for? And why 2?

Reply:
Oh I see, because of vdiffr expect_doppleganger. Could be a slightly better name :)